src/starwars/components/characterDetails/CharacterDetails.tsx

```diff
@@ -0,0 +1,34 @@
+import "./CharacterDetails.css";
+import { handleShare } from "../../../common/services/SharingService";
+
+interface CharacterDetailsProps {
```

Comment:
let's maby change it a littile bit. I don't think having every prop optional is a good way of doing stuff. Make then reqiured and in place of use do sth like: `character ? <SwCharatcet name={character.name} ... /> : <>No data<>`